2022q4/portmgr.adoc

```diff
@@ -0,0 +1,67 @@
+=== Ports Collection
+
+Links: +
+link:https://www.FreeBSD.org/ports/[About FreeBSD Ports] URL:link:https://www.FreeBSD.org/ports/[https://www.FreeBSD.org/ports/] +
+link:https://docs.freebsd.org/en/articles/contributing/#ports-contributing[Contributing to Ports] URL: link:https://docs.freebsd.org/en/articles/contributing/#ports-contributing[https://docs.freebsd.org/en/articles/contributing/#ports-contributing] +
+link:http://portsmon.freebsd.org/[FreeBSD Ports Monitoring] URL: link:http://portsmon.freebsd.org/[http://portsmon.freebsd.org/] +
+link:https://www.freebsd.org/portmgr/[Ports Management Team] URL: link:https://www.freebsd.org/portmgr/[https://www.freebsd.org/portmgr/] +
+link:http://ftp.freebsd.org/pub/FreeBSD/ports/ports/[Ports Tarball] URL: link:http://ftp.freebsd.org/pub/FreeBSD/ports/ports/[http://ftp.freebsd.org/pub/FreeBSD/ports/ports/]
+
+Contact: René Ladan <portmgr-secretary@FreeBSD.org> +
+Contact: FreeBSD Ports Management Team <portmgr@FreeBSD.org>
+
+The Ports Management Team is responsible for overseeing the overall direction of the Ports Tree, building packages (through its subsidiary pkgmgr), and personnel matters.
+Below is what happened in the last quarter.
+
+Currently we have just over 31,000 ports in the Ports Tree.
+There are currenly close to 2900 open ports PRs of which almost 800 are unassigned.
+The last quarter saw 8194 commits by 159 committers on the main branch and 657 commits by 53 committers on the 2022Q4 branch.
+Compared to the quarter before, this means a small increase in the number of available ports but also in the number of open PRs and a decreasing number of commits made.
+
+On the personnel front, we welcomed Ronald Klop (ronald@) and said goodbye to bar@ and bhughes@
+We welcomed pizzamig@ as a new official member after a succesful lurking period.
+We also welcomed three new lurkers: bofh@, ler@, and ygy@.
+
+Portmgr split itself up into portmgr and pkgmgr.
+The new pkgmgr team, currently consisting of anoine@ and bdrewery@, is responsible for building packages and maintaining the package building cluster.
+
+Four new USES were introduced:
+* llvm to canocalize ports dependencies on LLVM
+* luajit to select a LuaJIT runtime
+* octave to help ports to depend on Octave and Octave-Forge
```

Comment:
s/to depend/depend/ (or maybe "that depend").

Reply:
"depend"  it is